specs/CriticalRestartRequired.md

```diff
@@ -0,0 +1,105 @@
+Critical Update Available
+===
+
+# Background
+As WebView2 developers, we often have to author ECS (experimentation and configuration service - 
+used to perform A/B testing or remotely disable features) configurations to toggle feature flags. 
```

Comment:
A/B testing doesn't sound "critical" to me. Nothing goes *wrong* if the app doesn't restart their webviews.

Reply:
Yes. That's trying to define what ECS is. Not all ECS configuration updates are to solve critical issues.

Reply:
Do we want to talk about ECS in the public documentation?

Reply:
I was trying to avoid it that's why its only mentioned in the background. I thought it would be sort of confusing to introduce in the ref docs and we don't want to limit the event to just ECS